packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/StockEvolution.tsx

```diff
@@ -72,7 +48,7 @@ export const StockEvolution: React.FC<StockEvolutionProps> = ({
     }
   };
 
-  const tooltipLabelFormatter = (date: Date) => d(date);
+  const tooltipLabelFormatter = (date: string) => d(new Date(date));
```

Comment:
I'm seeing US-style dates (Month/Day/Year) in the tooltip, not sure if related to changes here?
![Screen Shot 2022-03-28 at 2 58 58 PM](https://user-images.githubusercontent.com/5456533/160313895-47171c38-401e-4cca-9061-e02cfdf61199.png)

Reply:
oh, well spotted. I was parsing dates when remapping the data - am still parsing the NaiveDate format, so maybe it was a problem all along. will sort it.

Reply:
ugh. existing problem with `i18n` date formatting. I've given up on it - can't see a way of not being a US date default, so have instead implemented using standard intl date formatting, using the browser locale as the seed. thanks for catching it!